meta-iot2000-example/kas.yml

```diff
@@ -0,0 +1,23 @@
+header:
+  version: 2
+  includes:
+    - ../meta-iot2000-bsp/kas.yml
```

Comment:
I prefer the syntax:

    - file: meta-iot2000-bsp/kas.yml
      repo: meta-iot2000

in this case more.

This way this file would be independent from its location in this repository. I don't really like it if includes reference files from a parent directory. Referencing files this way that are under the current directory or its sub-directories is fine.